explainaboard/analysis/analyses.py

```diff
@@ -318,7 +321,7 @@ class ComboCountAnalysisResult(AnalysisResult):
     """
 
     features: tuple[str, ...]
-    combo_counts: list[tuple[tuple[str, ...], int]]
+    combo_counts: list[tuple[tuple[str, ...], int, list[int]]]
```

Comment:
1. This variable looks no longer trivial. I think introducing a frozen dataclass is better.
2. The second member should be removed because it is fully transitive from `len(third_member)`.
```suggestion
    combo_counts: list[ComboOccurence]
```
where:
```python
@dataclass(frozen=True)
class ComboOccurence:
    """A struct representing occurences of the string tuples.

    Args:
        features: The feature values that the occurence is counted.
        sample_ids: List of sample IDs that contain the feature.
    """

    features: tuple[str, ...]
    sample_ids: list[int]
```

Reply:
I also guessed `dict[tuple[str, ...], list[int]]` is much better.